### PR TITLE
Closes #104 - merged state not listed in documentation

### LINKED ISSUE #104: Missing a ticket state
There is a state not mentioned in https://user-docs.zammad.org/en/latest/basics/service-ticket/settings/state.html
In German it is called "zusammengeführt", which would roughly translate to "merged".

### PR DESCRIPTION
**Pull request by MrGeneration**

### CODE REVIEW

File: basics/service-ticket/settings.rst [hunk 1]
```diff
@@ -48,6 +48,8 @@ text. (Your highlights are visible to other agents.)
 
 ----
 
+.. _ticket-submenu:
+
```
Comment (YetAnotherGerrit) [inline basics/service-ticket/settings.rst:52]: I don't understand the meaning of this change, but when I compare the old and thew new site, I recognize a change in the distance between the header "Highlighting ticket text" and the image above. The image sticks to the header below:
grafik

In my opinion it should look more like in the previous version:
grafik

**Review by YetAnotherGerrit — CHANGES_REQUESTED** [review]

File: basics/service-ticket/settings.rst (hunk 1 above)
Comment (MrGeneration) [reply basics/service-ticket/settings.rst:52]: This change depends on this reference:
https://github.com/zammad/zammad-user-documentation/pull/108/files#diff-7a8f49e5e4a29da5ba6c7ec877ae13a46872a511024759413cb76b6a18759077R13

I'll have a look on the formatting part later if the rest fits.

Comment (YetAnotherGerrit) [reply basics/service-ticket/settings.rst:52]: Yes, the rest fits.

Comment (MrGeneration) [reply basics/service-ticket/settings.rst:52]: Just as a follow up: I can't reproduce what you're seeing.
I'll merge and check on the online version to ensure it's not a bug.

Possibly your environment is missing something or behaves differently due to OS differences.

**Pull request merged**